### ISSUE TITLE
Create an initial 'quality' STAC extension

### ISSUE THREAD

**Issue by billgeo** [linz/geostore#843]
### Enabler

So that we can understand the work in validating a full LINZ STAC extension(s), we want to create a STAC extension for validation of one metadata property

#### Acceptance Criteria

- [x] Profile created that we can use for automated validation in the Geostore
- [x] Use the process and templates from the STAC community
- [x] Profile can be used to validate the mandatory 'quality' extension. 
- [x] Includes optional`quality:horizontal_accuracy` property
- [x] Stretch goal to include the optional`quality:horizontal_accuracy_type` property (only two values allowed)
- [x] Stretch goal to add the non-mandatory property `quality:description` property
- [x] Add a README, tests and examples for the JSON schema

#### Additional context
 
For more information on the properties to implement (still under development) see here https://confluence.linz.govt.nz/display/LIFFP/STAC+Field+Mapping and here https://confluence.linz.govt.nz/display/LIFFP/Core+and+Spatial+Metadata+Content

#### Tasks

- [x] Use template for extension and use LINZ stac extensions repo
- [x] Create README
- [x] Create JSON Schema
- [x] Add tests (do we need to do this?)

#### Definition of Ready

- [ ] This story is **ready** to work on, according to the
      [team's definition](https://confluence.linz.govt.nz/pages/viewpage.action?pageId=87930423)

#### Definition of Done

- [ ] This story is **done**, according to the
      [team's definition](https://confluence.linz.govt.nz/pages/viewpage.action?pageId=87930423)

**Comment by billgeo** [comment]
This looks good @MitchellPaff and @l0b0, once https://github.com/linz/STAC/pull/5 is merged, this story can be closed.
